source/blog/2017-05-25-future-plans-for-fedora-atomic-release-life-cycle.html.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Future Plans for the Fedora Atomic Release Life Cycle
+author: jbrooks
+date: 2017-05-25 07:00:00 UTC
+tags: fedora
+comments: true
+published: true
+---
+
+The Fedora Project ships new releases at roughly six-month intervals, and [maintains](https://fedoraproject.org/wiki/Fedora_Release_Life_Cycle) each release for around thirteen months. So Fedora N is supported by the community until one month after the release of Fedora N+2.
+
+Since the first Fedora Atomic host shipped, as part of Fedora 21, the project has maintained separate ostree repositories for both of the active Fedora releases. For instance, there are currently trees available for Fedora Atomic 25 and Fedora Atomic 24.
+
+Fedora Atomic sets out to be a particularly fast-moving branch of Fedora, with releases every two weeks and updates to key “atomic” components such as docker and kubernetes that move more quickly than one might expect from Fedora.
+
+Due in part to this faster pace, the Fedora Atomic workgroup has always focused its testing and integration efforts most directly on the latest stable release, encouraging users of the older release to rebase to the newer tree as soon as possible. Releases older than the current tree are supported only on a "best effort" basis, meaning that the ostree is updated, but there is no organized testing of the older releases.
+
+Starting with either the Fedora 26 to 27 or the 27 to 28 upgrade cycle, the Fedora Atomic Workgroup intends to collapse Fedora Atomic into a single version which will track the latest stable Fedora branch. When a new stable version of Fedora is released, Fedora Atomic users will automatically shift to the new version when they install updates.
```

Comment:
yes - we just have different refs for the different branches in a single ostree repo. This is really the only way to do it since we essentially have to start writing in f27 to test it and get it ready for release while we still support f26.